frontend/screens/LoginPage.js

```diff
@@ -1,136 +1,183 @@
 import * as React from "react";
 import { Image } from "expo-image";
-import { StyleSheet, View, Text, TouchableOpacity, ScrollView } from "react-native";
+import { StyleSheet, View, Text, TouchableOpacity, ScrollView, TextInput, Alert } from "react-native";
 import { useNavigation } from "@react-navigation/native";
 
-import Depth1Frame6 from "../components/Depth1Frame6";
 import { Color, Padding, FontSize, Border, FontFamily } from "../GlobalStyles";
 
 const LoginPage = () => {
   const navigation = useNavigation();
 
+  const [username, setUsername] = React.useState("");
+  const [password, setPassword] = React.useState("");
+
+  //It handle the login functionality
+  const handleInfluencerLogin = async () => {
+    if (username.trim() != "" && password.trim() != "") {
+      try {
+        const response = await fetch(`${process.env.API_ENDPOINT}/influencers/login`, {
+          method: "post",
+          headers: {
+            "Content-Type": "application/json"
+          },
+          body: JSON.stringify({ username, password })
+        });
+        const data = await response.json()
+        if (response.status == 200){
+          Alert.alert("Success", data.message)
+          navigation.navigate("UserProfile")
+          setUsername("")
+          setPassword("")
+        }
+        else
+          Alert.alert("Error", data.message)
+      }
+      catch (error) {
+        Alert.alert("Error", error.message)
+      }
+    }
+    else
+      Alert.alert("Error", "Please provide username and password")
+  }
+
   return (
-    <ScrollView contentContainerStyle={styles.scrollViewContent}>
-      <View style={styles.loginpage}>
-        <View style={styles.depth0Frame0}>
-        <TouchableOpacity onPress={() => navigation.navigate('BrandorInfluencer')}>
+    <ScrollView style={{ width: '100%', height: "100%", paddingTop: 30, backgroundColor: Color.colorWhitesmoke_100 }}>
```

Comment:
can we create a scss class and define those classes under that file? Alsom it would great if we can have a `login` folder under `screens` containing these 2 login files and its scss file